src/cls/rbd/cls_rbd.cc

```diff
@@ -108,6 +108,8 @@ cls_method_handle_t h_metadata_set;
 cls_method_handle_t h_metadata_remove;
 cls_method_handle_t h_metadata_list;
 cls_method_handle_t h_metadata_get;
+cls_method_handle_t h_get_snapshot_limit;
```

Comment:
So you want set_snapshot_limit and snapshot_limit? Since “limit” is a verb, “snapshot_limit” implies “set” to me. Your call, though.